.gitignore

```diff
@@ -44,7 +44,9 @@ docs/build
 docs/source/_tmp/*.md
 docs/source/command-ref/*.adoc
 docs/source/**/variables.adoc
-docs/source/using/images/minishift-architecture*
+docs/source/using/images/*
 docs/.bundle/
 docs/.sass-cache/
 docs/.java
+docs/source/using/.asciidoctor
+docs/?/
```

Comment:
This is something we need to fix, not just ignore. We need to find out what creates this file.